generated-migrations/PropelMigration_1726469485.php

```diff
@@ -0,0 +1,274 @@
+<?php
+use Propel\Generator\Manager\MigrationManager;
+
+/**
+ * Data object containing the SQL and PHP code to migrate the database
+ * up to version 1726469485.
+ * Generated on 2024-09-16 06:51:25 by GUL */
+class PropelMigration_1726469485{
+    /**
+     * @var string
+     */
+    public $comment = '';
+
+    /**
+     * @param \Propel\Generator\Manager\MigrationManager $manager
+     *
+     * @return null|false|void
+     */
+    public function preUp(MigrationManager $manager)
+    {
+        // add the pre-migration code here
+    }
+
+    /**
+     * @param \Propel\Generator\Manager\MigrationManager $manager
+     *
+     * @return null|false|void
+     */
+    public function postUp(MigrationManager $manager)
+    {
+        // add the post-migration code here
+    }
+
+    /**
+     * @param \Propel\Generator\Manager\MigrationManager $manager
+     *
+     * @return null|false|void
+     */
+    public function preDown(MigrationManager $manager)
+    {
+        // add the pre-migration code here
+    }
+
+    /**
+     * @param \Propel\Generator\Manager\MigrationManager $manager
+     *
+     * @return null|false|void
+     */
+    public function postDown(MigrationManager $manager)
+    {
+        // add the post-migration code here
+    }
+
+    /**
+     * Get the SQL statements for the Up migration
+     *
+     * @return array list of the SQL strings to execute for the Up migration
+     *               the keys being the datasources
+     */
+    public function getUpSQL(): array
+    {
+        $connection_default = <<< 'EOT'
+INSERT IGNORE INTO langs (lang_id, lang_iso_639-1, lang_iso_639-2, lang_iso_639-3, lang_name, lang_name_original, lang_created, lang_updated) VALUES ('1', 'aa', 'aar', '', 'Afar', 'Afaraf', NULL, NULL);
```

Comment:
**issue:** En lançant la migration, j'ai une erreur :

>  SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL  
   server version for the right syntax to use near '-1, lang_iso_639-2, lang_iso_639-3, lang_name, lang_name_original, lang_created,' at line 1

Je pense que c'est dû au `-` dans le nom de la colonne. En mettant les noms de colonnes entre backticks, ça marche :

```suggestion
INSERT IGNORE INTO langs (lang_id, `lang_iso_639-1`, `lang_iso_639-2`, `lang_iso_639-3`, lang_name, lang_name_original, lang_created, lang_updated) VALUES ('1', 'aa', 'aar', '', 'Afar', 'Afaraf', NULL, NULL);
```

Je ne sais pas pourquoi ça passe dans la CI par contre.

Reply:
Bien vu, je corrige !